app/views/mentee_application_mailer/notify_for_rejection.html.erb

```diff
@@ -0,0 +1,16 @@
+
+<div style='display: block; max-width: 36rem; margin: 0 auto; padding-top: 4rem;'>
+  <p><%= @recipient.full_name %>,</p>
+
+  <p>
+    Thank you for submitting your application to join us at the <strong>Agency of Learning</strong>!
+  </p>
+
+  <p>
+    Unfortunately, we got more applicants than we could admit in this current cohort. Though we know this can feel discouraging, we strongly encourage you to <strong>reapply for the next cohort</strong>. We take into consideration the progress you've made since your last application.
```

Comment:
I have already copy edited this somewhere, this is old copy

Reply:
we do not want to tell them how to feel about this, let alone instruct them that they should feel discouraged!

Reply:
Thank you for submitting your application to join us at the Agency of Learning!

Unfortunately you have not been accepted into the upcoming batch. This was a tough decision. We can't make any promises about being admitted on a future application, but we hope that you reapply for a future batch.

Reply:
sounds good, thanks for the help with the copy.